client/team.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2023 Kondukto
+
+*/
+
+package client
+
+import "net/http"
+
+type Team struct {
+	Name             string           `json:"name"`
+	IssueResponsible IssueResponsible `json:"issue_responsible"`
+}
+
+type IssueResponsible struct {
+	Username string `json:"username"`
+}
+
+func (c *Client) CreateTeam(teamName, responsible string) error {
+	var team = Team{
+		Name: teamName,
+		IssueResponsible: IssueResponsible{
+			Username: responsible,
+		},
+	}
+
+	req, err := c.newRequest(http.MethodPost, "/api/v3/teams", team)
+	if err != nil {
+		return err
+	}
+
+	_, err = c.do(req, nil)
```

Comment:
Just I think. why don't you check the status code?

Reply:
Great question. 🍀 
Most of kdt commands doesn't control the status code. Most important part is "Is There a error occured when make a request?. If not, everything is OK. But not, mark as fail current operation. Do not handle errors"

We'll improve the kdt in further.